pkg/cmd/extension/manager.go

```diff
@@ -249,6 +250,64 @@ func (m *Manager) installDir() string {
 	return filepath.Join(m.dataDir(), "extensions")
 }
 
+func (m *Manager) Create(name string) error {
+	exe, err := m.lookPath("git")
+	if err != nil {
+		return err
+	}
+
+	err = os.Mkdir(name, 0755)
+	if err != nil {
+		return err
+	}
+
+	initCmd := m.newCommand(exe, "init", "-b", "main", "--quiet", name)
+	err = initCmd.Run()
+	if err != nil {
+		return err
+	}
+
+	fileTmpl := heredoc.Docf(`
+		#!/bin/bash
+		set -e
+		echo "Hello %[1]s"
+
+		# Examples to help get started:
+
+		# Determine if an executable is in the PATH
+		# if [ "$(which go)" = "" ]; then
+		#   echo "go not found in PATH"
+		#   exit 1
+		# fi
+
+		# Pass arguments through to a new process
+		# exec gh issue list "$@" -R cli/cli
```

Comment:
I'm not sure if we should show `exec` as an example in the middle of the script. If someone copy-pastes this, nothing after `exec` will run as exec will replace the current process with the subprocess. Since `exec` should always be the last process spawned, but is not strictly needed here, I would just omit it
```suggestion
		# gh issue list "$@" -R cli/cli
```